fido-device-onboard.spec

```diff
@@ -0,0 +1,122 @@
+%define dracutlibdir %{_prefix}/lib/dracut
+%bcond_without check
+%global __cargo_skip_build 0
+%global __cargo_is_lib() false
+%global gitversion 43f7674dad4abc470cdab505fbf49f571539cea8
+
+Name:           fido-device-onboard
+Version:        0.1.0
+Release:        1%{?dist}
+Summary:        An implementation of the FIDO Device Onboard Specification written in rust
+
+License:        BSD 3
+URL:            https://github.com/fedora-iot/fido-device-onboard-rs/
+Source:         %{url}/archive/%{gitversion}/%{name}-rs-%{gitversion}.tar.gz
+Source1:        %{name}-rs-%{gitversion}-vendor.tar.gz
```

Comment:
Given that Fedora doesn't vendor things, maybe make this Source dependent on the distro it's being built for?

Reply:
work to make this spec working on fedora is still ongoing but I don't want to block rhel packaging on that

Reply:
it's now rhel only but we still need to find a way to add buildrequires for fedora